sound/soc/sof/loader.c

```diff
@@ -192,6 +192,31 @@ static int ext_man_get_dbg_abi_info(struct snd_sof_dev *sdev,
 	return 0;
 }
 
+static int ext_man_get_config_data(struct snd_sof_dev *sdev,
+				    const struct sof_ext_man_elem_header *hdr)
+{
+	const struct sof_ext_man_config_data *config_data =
+		container_of(hdr, struct sof_ext_man_config_data, hdr);
+	int i;
+
+	/* TODO: Currently CAVS_LPRO only supported */
```

Comment:
@fredoh9 do we really need this TODO? its quite self-explanatory depending on the token types defined isnt it?